src/views/ControlPanel.vue

```diff
@@ -413,6 +409,28 @@ export default {
 					if (!confirm || confirm !== 'yes') {
 						return
 					}
+				} else if (action === 'beginHealingNetwork') {
+					const { includeSleeping } =
+						await this.$listeners.showConfirm(
+							'Info',
+							'Healing network causes a lot of traffic, can take minutes up to hours and users have to expect degraded performance while it is going on',
+							'info',
+							{
+								confirmText: 'Heal',
+								inputs: [
+									{
+										type: 'checkbox',
+										label: 'Include sleeping nodes',
+										key: 'includeSleeping',
+										value: false,
```

Comment:
Default is `true` in the driver for compatibility reasons. I guess default `false` here is okay.